reward/staking_manager.go

```diff
@@ -42,17 +43,24 @@ type blockChain interface {
 type StakingManager struct {
 	ac           *addressBookConnector
 	sic          *stakingInfoCache
+	sidb         *stakingInfoDB
 	gh           governanceHelper
 	bc           blockChain
 	chainHeadCh  chan blockchain.ChainHeadEvent
 	chainHeadSub event.Subscription
 	isActivated  bool
 }
 
-func NewStakingManager(bc blockChain, gh governanceHelper) *StakingManager {
+func NewStakingManager(bc blockChain, gh governanceHelper, dbm database.DBManager) *StakingManager {
+	sidb := &stakingInfoDB{}
+	if dbm != nil {
+		sidb.db = dbm.GetStakingInfoDB()
+	}
+
 	return &StakingManager{
 		ac:          newAddressBookConnector(bc, gh),
 		sic:         newStakingInfoCache(),
```

Comment:
I think we need to rename all variables in `StakingManger` like below.
`ac` -> `addressBookConnector
`sic` -> `stakingInfoCache`
...

What do you think? @winnie-byun @aidan-kwon

Reply:
I agree. How about like this?
```
	return &StakingManager{
		addressBookConnector: newAddressBookConnector(bc, gh),
		stakingInfoCache:     newStakingInfoCache(),
		stakingInfoDB:        sidb,
		governanceHelper:     gh,
		blockchain:           bc,
		chainHeadChan:        make(chan blockchain.ChainHeadEvent, chainHeadChanSize),
		isActivated:          false,
	}
```

Reply:
Good. :)
You can do this in this PR or another PR.

Reply:
It is now reflected. Thank you.